apps/trading/components/rewards-container/streaks/activity-streaks.spec.tsx

```diff
@@ -1,6 +1,17 @@
 import { render, screen } from '@testing-library/react';
 import { ActivityStreak } from './activity-streaks';
 
+jest.mock('@vegaprotocol/network-parameters', () => ({
+  ...jest.requireActual('@vegaprotocol/network-parameters'),
+  useNetworkParams: jest.fn(() => ({
+    params: {
+      rewards_activityStreak_inactivityLimit: '3.00',
```

Comment:
I don't think 3.00 is valid?
At least in my testing it doesn't get accepted.

Reply:
i got it from https://explorer.vega.xyz/network-parameters 
it should be, right?
<img width="1629" alt="image" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/f73df1ef-8fc8-4ae3-b2a1-c931deee9cbb">